### PR TITLE
documenting undocumented class methods

### PR DESCRIPTION
**Pull request by tiagmoraismorgado**

### CODE REVIEW

**Comment by snappizz** [comment]
nope, sorry. these additions are practically free of content, are full of typos, and you misplaced them in the "Asynchronous Commands" section so that now part of the docs for `sync` are part of `aliveThreadPeriod`.

**Comment by tiagmoraismorgado** [comment]
would there be a chance to fix this, at least?

**Comment by tiagmoraismorgado** [comment]
```
method:: name 
Returns specified server name 

method:: isRecording 
Returns whether server is recording 

method:: aliveThreadIsRunning 
Returns whether alive thread is running 

method:: aliveThreadPeriod 
Returns a value with preceding alive thread period
```

@snappizz here is an attempt to fix some of the typos. i would appreciate if you would collaborate in the process, of getting the fixes into the right place. with a bit of revision this can be perfectly merged IMHO, so please collaborate. in the end what we all want is making supercollider better

**Comment by telephon** [comment]
@tiagmoraismorgado thank you! But it's much easier if you just put them where they belong, instead of asking others who are working on something else at the moment.

**Comment by brianlheim** [comment]
@tiagmoraismorgado I agree with @telephon, these changes are easy enough that you can do them on your own.

**Comment by tiagmoraismorgado** [comment]
i will do so

**Comment by tiagmoraismorgado** [comment]
guys, can you please open the PR again so that i can fix the suggested improvements and who knows, adding a few further stuff? kind regards. Tiago

**Comment by snappizz** [comment]
github lets you push to the branch while the PR is closed.

**Comment by tiagmoraismorgado** [comment]
so, how can i make the edit button show up?
captura de ecra 2017-06-05 as 18 39 00

**Comment by bagong** [comment]
What you're seeing is a diff. Github doesn't expect you to edit diffs, it autogenerates them from changes made in actual documents. Press view and then look out for the pencil in the upper right corner.

**Comment by jamshark70** [comment]
> so, how can i make the edit button show up?

If you expect to contribute regularly, it's really worth it to install git on your system and create the changes in your local repository. One of the big benefits is, if you make a mistake with some commits, you can fix them locally before pushing and the public repository never sees your errors ;) I like that feature.

The pencil button is good for one-off, small changes, but it isn't ideal as a long-term workflow. You really need git locally.

(A few years ago, we were on SVN, and nobody could contribute *anything* without using an svn client on their own machine... no web edits at all. So I am not suggesting anything unrealistic here. Git is the tool we use... learn it.)

File: HelpSource/Classes/Server.schelp [hunk 1]
```diff
@@ -672,6 +672,18 @@ argument:: bundles
...
+Returns specified server name
+
+method:: isRecording
+Returns wether server is recording
+
+method:: aliveThreadIsRunning
+Returns wether alive thread is running
+
+method:: aliveThreadPeriod
+Returns a value with predeceasing alive thread period
```
Comment (vivid-synth) [inline HelpSource/Classes/Server.schelp]: I don't understand what this means -- can you clarify?

**Pull request closed without merging**